guide/guide/.js/src/main/scala/io/udash/web/guide/views/frontend/demos/DateTimeLocalDemo.scala

```diff
@@ -34,17 +34,16 @@ object DateTimeLocalDemo extends AutoDemo {
     )
 
     input.render
-
     (input, input)
   }.withSourceCode
 
-  override protected def demoWithSource(): (Modifier, Iterator[String]) = {
+  override protected def demoWithSource(): (Modifier, String) = {
     import io.udash.bootstrap.utils.BootstrapStyles._
     import io.udash.css.CssView._
     (div(id := "datetimelocal-input-demo", GuideStyles.frame, GuideStyles.useBootstrap)(
       form(containerFluid)(
         firstInput, br, secondInput
       )
-    ), source.linesIterator.take(source.linesIterator.size - 2))
+    ), source.linesWithSeparators.toList.dropRight(1).mkString)
```

Comment:
maybe some util `AutoDemo` method for that

Reply:
Can you suggest anything? I don't want to hardcode `2` there when it's dependent on the actual source code.

Reply:
I mean this `dropRight(1)` has a specific purpose(removing return part prolly) and is used in multiple places? Let's just create metod `stripReturnStatement` or sth in autodemo and use it.